rust/kcl-lib/src/std/args.rs

```diff
@@ -182,13 +182,22 @@ impl Args {
             }))?;
 
         T::from_kcl_val(&arg.value).ok_or_else(|| {
+            let expected_type_name = tynm::type_name::<T>();
```

Comment:
Rather than add a dep, could we just `rfind("::")` and substring? I don't usually mind adding deps, but this seems pretty easy to do ourselves and I hope we can stop using Rust types for anything user-facing in the short to medium term future too

Reply:
Totally open to it, but, it's a very light dependency, its only dependency is nom, and I like that it can parse traits etc straightforwardly.

I'd rather stop exposing Rust types directly and instead use a trait to give Rust types a KCL human-friendly name... hmmm... let me try that.